plyer/platforms/win/libs/win_api_defs.py

```diff
@@ -9,11 +9,10 @@
            'DestroyWindow', 'LoadIconW')
 
 import ctypes
-from ctypes import (Structure, windll, sizeof, byref, POINTER, memset,
-                    WINFUNCTYPE)
+from ctypes import Structure, windll, sizeof, POINTER, WINFUNCTYPE
 from ctypes.wintypes import (DWORD, HICON, HWND, UINT, WCHAR, WORD, BYTE,
-    LPCWSTR, LPWSTR, INT, LPVOID, HINSTANCE, HMENU, LPARAM, WPARAM,
```

Comment:
I think we can leave these ones here for this is a lib and maybe useful later...